Xerpa/tenjin
# Adds new endpoint for HOCR

### Description:

In preparation for the [Salary Plots](https://xerpabr.atlassian.net/browse/SSOT-1490), this PR adds a new endpoint that uses HOCR to parse a paystub table and returns a JSON with the parsed information, line by line.

### How to test:

1. Start Tenjin.
2. Send to `/hocr` a payload: its body should be the cropped table from a paystub, and it should receive as a `columns` argument a JSON with the column names and their bounding boxes. For example:
```
{"código":{"x_0":18,"x_1":282,"y_0":105,"y_1":1212},
"descontos":{"x_0":2805,"x_1":3534,"y_0":99,"y_1":1209},
"descrição":{"x_0":300,"x_1":1674,"y_0":99,"y_1":1209},
"referência":{"x_0":1692,"x_1":2145,"y_0":99,"y_1":1212},
"vencimentos":{"x_0":2157,"x_1":2793,"y_0":99,"y_1":1215}}
```
3. It should return a JSON List with each line as a JSON Object with the required columns.
